examples/Vanilla/Conversion/index.html

```diff
@@ -36,21 +36,17 @@ <h1>Conversion</h1>
         Next
       </button>
     </form>
-
-        <script src="./../../../dist/rci-legacy.min.js"></script>
-<!--        <script src="../../../dist/rci.min.js"></script>-->
-<!--    <script src="https://sdk.real-user-data.eggplant.cloud/latest/rci.min.js"></script>-->
-
-
-    <!--
-    Make sure that you run one of the following 2 commands:
-    webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/Conversion/conversion.js  --scope implementation --rulesTarget legacy --mode development
-    webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/Conversion/conversion.js  --scope implementation --rulesTarget current --mode development
-
-    and then uncomment the appropriate one
+    <!--  Make sure that you run one of the following 2 commands and then uncomment the appropriate one below
+        webpack --config=config/webpack.js --entryPoint ./examples/Vanilla/CustomCollector/customCollector.js  --scope implementation --rulesTarget legacy --mode development
```

Comment:
shouldn't the commands point to Conversion directory?

Reply:
ok fixed